packages/api/test/codegen/languages/typescript.test.ts

```diff
@@ -197,8 +196,8 @@ describe('typescript', () => {
           });
       });
 
-      it('should be able to make an API request (JS + CommonJS)', async () => {
-        const sdk = await import('../../__fixtures__/sdk/simple-js-cjs').then(r => r.default);
+      it.skip('should be able to make an API request (JS + CommonJS)', async () => {
```

Comment:
These tests that are testing CJS built SDKs no longer work because of the way that these modules are handled now. These tests will get reworked in https://github.com/readmeio/api/pull/734 when we refactor the codegen process to _properly_ build CJS exports.